src/py42/services/auditlogs.py

```diff
@@ -0,0 +1,67 @@
+from py42 import settings
+from py42.services import BaseService
+from py42.services.util import get_all_pages
+from py42.util import convert_timestamp_to_str
+from py42.util import to_list
+
+
+class AuditLogsService(BaseService):
+    def get_page(self, page_num, page_size=None, **kwargs):
+
+        event_types = to_list(kwargs.pop(u"event_types"))
+        user_ids = to_list(kwargs.pop(u"user_ids"))
+        usernames = to_list(kwargs.pop(u"usernames"))
+        user_ip_addresses = to_list(kwargs.pop(u"user_ip_addresses"))
```

Comment:
Should these list types default to empty list when not provided in `kwargs`?  Not a big deal since this is internal, but would be safer.

Reply:
agreed

Reply:
ummm changed.